internal/server/index.go

```diff
@@ -14,9 +15,40 @@ import (
 
 var alphaNumericRunes = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
 
-// ServeIndex serve the index.html file with nonce generated for supporting
-// restrictive CSP while using material-ui.
-func ServeIndex(publicDir string) fasthttp.RequestHandler {
+// ServeEmbeddedIndex serve the index.html file with nonce generated for supporting
+// restrictive CSP while using material-ui from the embedded virtual filesystem.
+func ServeEmbeddedIndex(publicDir string) fasthttp.RequestHandler {
+	f, err := br.Open(publicDir + "/index.html")
+	if err != nil {
+		logging.Logger().Fatalf("Unable to open index.html: %v", err)
```

Comment:
we should write an error response and return from the function here. Can you also fix it in the other handler (I forgot to do it when I introduced it).